scripts/deploy.sh

```diff
@@ -67,6 +70,7 @@ fi
 which openstack-ansible || ./scripts/bootstrap-ansible.sh
 
 # ensure all needed passwords and tokens are generated
+./scripts/pw-token-gen.py --file /etc/openstack_deploy/user_secrets.yml
```

Comment:
@mattt416 - it's not that I don't think it is needed, it that it isn't already needed.